stakingModel/test/GoodReserveCDai.test.js

```diff
@@ -332,33 +363,29 @@ contract("GoodReserve - staking with cDAI mocks", ([founder, staker]) => {
     let reserveToken = await marketMaker.reserveTokens(cDAI.address);
     let reserveBalanceBefore = reserveToken.reserveSupply;
     let supplyBefore = reserveToken.gdSupply;
-    let rrBefore = reserveToken.reserveRatio;
     const gdBalanceBefore = await goodDollar.balanceOf(founder);
     const cDAIBalanceBefore = await cDAI.balanceOf(founder);
     const cDAIBalanceReserveBefore = await cDAI.balanceOf(goodReserve.address);
-    const priceBefore = await goodReserve.currentPrice(cDAI.address);
     await goodDollar.approve(goodReserve.address, amount);
     let transaction = await goodReserve.sell(cDAI.address, amount, 0);
     reserveToken = await marketMaker.reserveTokens(cDAI.address);
     let reserveBalanceAfter = reserveToken.reserveSupply;
     let supplyAfter = reserveToken.gdSupply;
-    let rrAfter = reserveToken.reserveRatio;
     const gdBalanceAfter = await goodDollar.balanceOf(founder);
     const cDAIBalanceAfter = await cDAI.balanceOf(founder);
     const cDAIBalanceReserveAfter = await cDAI.balanceOf(goodReserve.address);
-    const priceAfter = await goodReserve.currentPrice(cDAI.address);
+    // according to the initialization settings we should expect 1000000
```

Comment:
this says nothing. it is exactly what the test itself says "we are expecting 100000" but you are not explaining why 100000
how did you reach this number?

Reply:
d94eca8064aa0ed7e41c368365fad416f731edc5